hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -50,17 +50,17 @@
     {
       name: data_toggle_clear
       desc: '''
-            Verify the data toggle clear functionality.
+            Verify the data toggle restore functionality.
 
             - Set any random bit of the data toggle register from bit 0 and 11 then
               initiate an IN/OUT transaction on the corresponding endpoint whose bit is asserted.
-            - Verify that by writing 1 to particular bit (0-11 bits) will clear the data toggle bit for
-              that particular endpoint to DATA0 in IN/OUT directions to resynchronize with the USB host.
+            - Verify that writing 0 to a particular bit (bit 0-11 bit) resets the data toggle bit for
```

Comment:
I'm really sorry to be nitty, but do you mean "bit 0 - bit 11"? (Rather than "the bit numbered zero up to 11 bits in total"!)

Reply:
Sorted